scripts/clinical-clusters/build-clinical-clusters

```diff
@@ -1,11 +1,12 @@
 #!/usr/bin/env bash
 
+currentPerformanceYear=2018
```

Comment:
No problem if this is what makes sense here, but I think in other files we pass the perfYear as an argument to the build script-- is it better to just set it directly here?

Reply:
Gotcha, thank you!